.github/workflows/run_system_tests.yml

```
@@ -0,0 +1,21 @@
+name: Run system tests
+
+on:
+  workflow_call:
+  workflow_dispatch:
+jobs:
+  run_system_tests:
+    runs-on:
```

[cosmetic] Give this a name so GH doesn't display the snake_case id

```suggestion
  run_system_tests:
    name: Run system tests
    runs-on:
```